### PR TITLE
v3.2.0 的更新內容

### PR DESCRIPTION
**Pull request by dragonyc1002**
### Features
* Added some new info content. ([osu.js](./src/cmds/osu.js)) (5223334)

* Added `/lightsup` game. ([lightsUp.js](./src/cmds/lightsUp.js)) (76a9ca7)

* Added private `/sendann` command. ([supportGuildSendAnn.cjs](./src/cmds/lightsUp.js)) (524e951)

* Added clear commands script. ([clear.js](./src/clear.js)) (b6039b9)

### Changes
* Added clear script. ([package.json](package.json)) (4263563)

### Fixes
* Wrong typing in `/avatar` command. ([avatar.js](./src/cmds/avatar.js)) (a92adae)

### Refactors
* Created webhook object from a new file. ([WebhookManager.js](./src/utils/WebhookManager.js)) (a92adae)

* Import announcement content from a new file. ([lateatUpdate.js](./src/data/latestUpdate.js)) (e697451)

* Replaced `size` and `boardSize` value with `Number.prototype.toString()`  method. (ecf8f3a)

綜上，若有錯誤請告知我進行修改，若沒有即可直接 approve 。

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ See more at [LICENSE](./LICENSE) and [Team Of Service](./docs/TeamOfService.md)
 See at [Changelog](./CHANGELOG.md) documentation.
 
 ## Invite this bot
-See the invite link in [constants](./src/constants.json) file.
+You can invite this bot by clicking [this](https://discord.com/api/oauth2/authorize?client_id=977024737156931675&permissions=1644971949559&scope=bot%20applications.commands).
```
Comment (Mantou1233) [inline README.md]: a wording diff but i think [here] is more appropriate

File: config.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ if(!process.env.errHookToken) throw new Error('Error Log Webhook token has not b
 if(!process.env.reportHookId) throw new Error('Report Log Webhook id has not been configured.');
 if(!process.env.reportHookToken) throw new Error('Report Log Webhook token has not been configured.');
 
-if(!process.env.osuApiKey) console.log("No osu api key is configurated. This means the Osu command will not be usable.")
+if(!process.env.osuApiKey) throw new Error('Osu! Api Key has not been configured.');
```
Comment (Mantou1233) [inline config.js:16]: i still think this change should be reverted (because i dont have a osu api key)

File: docs/help.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 以下為指令說明。
 
 ## 聊天指令
 * /avatar - 查看一位使用者的頭像。
 
-* /echo - 強迫 HBYC 重複一句你說的話。(使用時請遵守[使用者條款](./TeamofService-Tw.md))
+* /echo - 強迫 HBYC 重複一句你說的話。(使用時請遵守[使用者條款](./TeamOfService-Tw.md))
 
-* /say - 強迫 HBYC 說出一句訊息。（使用時請遵守[使用者條款](./TeamofService-Tw.md))
+* /say - 強迫 HBYC 說出一句訊息。（使用時請遵守[使用者條款](./TeamOfService-Tw.md))
```
Comment (Mantou1233) [inline docs/help.md]: spelling mistakes, terms of service i think it is?

File: src/cmds/supportGuildSendAnn.cjs [hunk 4]
```diff
@@ -0,0 +1,31 @@
+// This is a private command that can send the message to the channel.
+const { SlashCommandBuilder } = require('@discordjs/builders');
+
+const { bot, supportGuild } = require('../constants.json');
+
+const config = require('../../config');
+const ann = require('../data/latestUpdate');
+
+const updateAnnData = new SlashCommandBuilder()
+  .setName('sendann')
```
Comment (Mantou1233) [inline src/cmds/supportGuildSendAnn.cjs]: well, uh this isnt a really great name "sendann". couldnt "annoucement" or such would be a better naming?

**Review by Mantou1233 — CHANGES_REQUESTED** [review]

**Comment by Mantou1233** [comment]
done the review. please check the few comments
also in WebhookManager, would namings such as "importWebhook" or "import-webhook" like naming would be better in usage?
also sorry for the bad english my chinese typings died

File: config.js (hunk 2 above)
Comment (dragonyc1002) [reply config.js:16]: i am planning to make a plugin of this, before i do the changes, i will still use `throw new Error`.

those changes may be out in next minor version.

Comment (Mantou1233) [reply config.js:16]: 👌

**Comment by dragonyc1002** [comment]
`WebhookManager` i think it's enough, and i'm lazy to change a different name.

**Comment by Mantou1233** [comment]
alright

**Pull request merged**